assets/js/util/refresh-authentication.js

```diff
@@ -23,7 +23,9 @@ import data, { TYPE_CORE } from '../components/data';
 
 export const refreshAuthentication = async () => {
 	try {
-		const response = await data.get( TYPE_CORE, 'user', 'authentication' );
+		// `timestamp` added to ensure this request is always made as it is preloaded
+		// using apiFetch's preloading middleware.
+		const response = await data.get( TYPE_CORE, 'user', 'authentication', { timestamp: Date.now() } );
```

Comment:
Just question about the internals of the `apiFetch` preloading middleware, is it that by passing `timestamp` specifically we can bypass it or is it just any parameters that would bypass it?